programs/drift/src/state/user.rs

```diff
@@ -546,6 +549,118 @@ impl User {
     }
 }
 
+pub const RFQ_PDA_SEED: &str = "RFQ";
+#[zero_copy(unsafe)]
+#[derive(Default, Eq, PartialEq, Debug)]
+#[repr(C)]
+pub struct RFQOrderId {
```

Comment:
might stick this rfq stuff into its own file just cuz user file is already giant